config.toml

```diff
@@ -443,15 +443,15 @@ memo_prefix = ''
 
 # Specify packet sequences which should not be cleared, per channel.
 #
-# For each channel, specify a list of sequences which should not be cleared, eg.
+# For each channel, specify a list of sequences which should not be cleared. Acceptable value
+# include range of sequences with separator "..=", ".." or "-", eg.
 #
-#   excluded_sequences = [
-#     ['channel-0', [1, 2, 3]],
-#     ['channel-1', [4, 5, 6]]
-#  ]
+#   [chains.excluded_sequences]
+#   channel-0 = [1, 2, "3..7", 9, 11],
+#   channel-1 = [4, 5, 6],
```

Comment:
While I agree that the new syntax does look better than the former one, I would rather introduce breaking changes to the config at this stage.

Do you think we can support both options? Otherwise let's just revert to the former one.

Reply:
Unfortunately I think this was a mistake in the documentation, because if I run the `parse_valid_config()` test, https://github.com/informalsystems/hermes/blob/v1.9.0/crates/relayer/src/config.rs#L842, with the following config, it fails.

config:
<details>
[global]
log_level = 'error'

[mode]

[mode.clients]
enabled = true
refresh = true
misbehaviour = true

[mode.connections]
enabled = false

[mode.channels]
enabled = false

[mode.packets]
enabled = true
clear_interval = 100
clear_on_start = true
tx_confirmation = true
ics20_max_memo_size = { enabled = true, size = "32KiB" }
ics20_max_receiver_size = { enabled = true, size = "2KiB" }

[[chains]]
type = "CosmosSdk"
id = 'chain_A'
rpc_addr = 'http://127.0.0.1:26657'
grpc_addr = 'http://127.0.0.1:9090'
event_source = { mode = 'push', url = 'ws://localhost:26657/websocket', batch_delay = '500ms' }
rpc_timeout = '10s'
account_prefix = 'cosmos'
key_name = 'testkey'
store_prefix = 'ibc'
max_gas = 200000
gas_price = { price = 0.001, denom = 'stake' }
max_msg_num = 4
max_tx_size = 1048576
clock_drift = '5s'
trusting_period = '14days'
trust_threshold = { numerator = '1', denominator = '3' }
excluded_sequences = [
  ['channel-0', [1, 2, 3]],
  ['channel-1', [4, 5, 6]]
]

[chains.packet_filter]
policy = 'allow'
list = [
  ['ica*', '*'],
  ['transfer', 'channel-0'],
]

[[chains]]
type = "CosmosSdk"
id = 'chain_B'
rpc_addr = 'http://127.0.0.1:26557'
grpc_addr = 'http://127.0.0.1:9090'
event_source = { mode = 'push', url = 'ws://localhost:26557/websocket', batch_delay = '500ms' }
rpc_timeout = '10s'
account_prefix = 'cosmos'
key_name = 'testkey'
store_prefix = 'ibc'
gas_price = { price = 0.001, denom = 'stake' }
clock_drift = '5s'
trusting_period = '14days'
trust_threshold = { numerator = '1', denominator = '3' }
address_type = { derivation = 'ethermint', proto_type = { pk_type = '/injective.crypto.v1beta1.ethsecp256k1.PubKey' } }
</details>


error:
```
Caused by:
    TOML parse error at line 25, column 1
       |
    25 | [[chains]]
       | ^^^^^^^^^^
    invalid CosmosSdk config: invalid type: sequence, expected a map
```

Reply:
Oh right, perfect then!